README.md

```diff
@@ -312,6 +313,21 @@ Use the following command to run all tests, with the functional tests running ag
 
 Note that Testcontainers only starts if LocalStack is not running on port 4566.
 
+#### Running Tests in your own project without import these beans
```

Comment:
Just some questions on this title really... why does it only matter in your own project? Wouldn't you leave this config in anyway? Also what does "these beans" mean - can we be more explicit? So maybe this title should say something like "Running tests without any queues / Localstack"?

Reply:
All of the above is about running the tests for the library whereas this instruction is about running tests in the client project. Maybe this is not clear